bin/run-selenium-remote.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env sh
+set -e
+
+echo '    Downloading selenium'
+docker pull selenium/standalone-firefox:2.53.1
```

Comment:
It's quite comparable it seems, the build takes about 30 seconds longer. It looks like this time is needed to download the docker image.

I think this is the most cost effective way of testing a remote instance of Selenium. If we want to try a "real" remote then we probably have to go with a hosted service like Browserstack.